disco_aws_automation/disco_deploy.py

```diff
@@ -667,12 +667,16 @@ def test(self, dry_run=False, deployment_strategy=None):
         amis = self.get_test_amis()
         if len(amis):
             self.test_ami(random.choice(amis), dry_run, deployment_strategy)
+        else:
+            logging.info("No 'untested' AMIs found.")
 
     def update(self, dry_run=False, deployment_strategy=None):
         '''Updates a single autoscaling group with a newer AMI'''
         amis = self.get_update_amis()
         if len(amis):
             self.update_ami(random.choice(amis), dry_run, deployment_strategy)
+        else:
+            logging.info("No updated AMIs found.")
```

Comment:
Hmm, the comment in get_update_amis says "Returns list of AMIs that are ready to be deployed in production.  Hosts must be in the pipeline definition and marked as deployable and the AMI must be newer than the one currently running AMI for that host." And, the update_ami function appears to update the hostclass with the latest ami. From that perspective it seems like it's the function names that have misleading names. Seems like they should be get_new_amis or get_updated_amis, and update_hostclass_with_ami. But correct me if I'm wrong/misunderstanding something.